kikuchipy/projections/lambert_projection.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019-2020 The kikuchipy developers
+#
+# This file is part of kikuchipy.
+#
+# kikuchipy is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# kikuchipy is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with kikuchipy. If not, see <http://www.gnu.org/licenses/>.
+
+from typing import Union
+
+import numpy as np
+from orix.vector import Vector3d
+
+from kikuchipy.projections.gnomonic_projection import GnomonicProjection
+
+
+class LambertProjection:
+    """Lambert Projection of a vector"""
```

Comment:
We should always cite work we use or are influenced by, in this case it is particularly important for the reader to understand this implementation. The Callahan & De Graef (2013) paper is already in the documentation (https://kikuchipy.org/en/stable/reference.html?highlight=callahan#module-kikuchipy.io.plugins.emsoft_ebsd_master_pattern), so you can reference it as `Lambert projection of a vector [Callahan2013]_.`